projects/libredwg/llvmfuzz.options

```diff
@@ -1,2 +1,3 @@
 [libfuzzer]
 detect_leaks=0
+timeout=4000
```

Comment:
Why do you need this. We have a 65 sec timeout per testcase. I don't think per-testcase processing should take longer than that in normal circumstances. note that fuzzing itself per cycle goes for like 2.3hrs (8400 sec)

Reply:
Because that timeout was needed locally to pass normal testcases from my corpus, and locally the default libfuzzer timeout is 1000.

Reply:
Checked again. The timeout is seconds, not milliseconds!
That's of course far too much. I removed it You were right